src/SFA.DAS.AssessorService.Application.UnitTests/Handlers/BatchLogs/PrintedBatchLogHandlerTests/When_called_and_batch_does_not_exist.cs

```diff
@@ -14,42 +12,30 @@
 
 namespace SFA.DAS.AssessorService.Application.UnitTests.Handlers.BatchLogs.PrintedBatchLogHandlerTests
 {
-    public class When_called_and_batch_does_not_exist
+    public class When_called_and_batch_does_not_exist : PrintedBatchLogHandlerTestBase
     {
-        private Mock<IBatchLogQueryRepository> _batchLogQueryRepository;
-        private Mock<ICertificateRepository> _certificateRepository;
-        private Mock<IMediator> _mediator;
-        private Mock<ILogger<PrintedBatchLogHandler>> _logger;
-
-        private static int _batchNumber = 222;
-        private static DateTime _printedAt = DateTime.UtcNow;
-
         private BatchLog _batchLog = new BatchLog { Id = Guid.NewGuid(), BatchNumber = _batchNumber };
         private ValidationResponse _response = new ValidationResponse();
 
         [SetUp]
         public async Task Arrange()
         {
-            MappingBootstrapper.Initialize();
+            //Arrange
+            base.BaseArrange();
 
-            _batchLogQueryRepository = new Mock<IBatchLogQueryRepository>();
-            
+            _batchLogQueryRepository = new Mock<IBatchLogQueryRepository>();            
             _batchLogQueryRepository.Setup(r => r.GetForBatchNumber(It.IsIn(_batchNumber))).Returns(Task.FromResult(_batchLog));
```

Comment:
This should be able to be moved into the new base in some form

Reply:
done